src/components/App/App.js

```diff
@@ -5,17 +5,23 @@ import Discover from '../Discover/Discover';
 import Collections from '../Collections/Collections';
 import Journal from '../Journal/Journal';
 import EmptyState from '../EmptyState/EmptyState';
+import Search from '../Search/Search'
+import Results from '../Results/Results'
+import Album from '../Album/Album'
 
 function App() {
   return (
     <>
       <Nav />
+      <Search/>
       <main className="App">
         <Routes>
           <Route path="/" />
           <Route path="/journal" element={<Journal />} />
           <Route path="/collections" element={<Collections />} />
           <Route path="/discover" element={<Discover />} />
+          <Route path='/search/:query/:page' element={<Results/>}/>
+          <Route path='/albums/:id' element={<Album/>}/>
```

Comment:
Love all the work you did with pagination to handle for this route, but I'd like some discussion on whether we need/want individual routes for each page.  The only benefit I see is it allowing use of the back/forward browser buttons - wouldn't it be easier if we just didn't route this stuff and expect the user to only use the provided `prev`/`next` buttons?  It would def make for cleaner code in our `Results` component.

@jmartin777 @jalbe0076 would love ya'lls thoughts too on this

Reply:
I like the idea of being able to use the forward and back buttons. I personally use it all the time and opt for it rather than the back button on a page.

Reply:
I tried that first but I was running into an error where the app broke when refreshing.
Basically you need the page in the url because if you're on a specific page and refresh, it breaks.  It seems like this is the way it's done in the real world, if you look at the url when going through pages of results on websites it's usually in the url somehow.  By using useParams on load of the url to get page and query, the api call sets results and pagination information and the state is all in the results file.  Before I was passing around setter functions and states, and now it's all in results, so it makes it cleaner in that way.

Reply:
Okay great! Thanks for the discussion yall and for putting all the thought and experimentation into this Alec - pagination is always tricky!